src/test/javascript/spec/component/tutorial-groups/helpers/tutorialGroupExampleModels.ts

```diff
@@ -0,0 +1,64 @@
+import { TutorialGroup } from 'app/entities/tutorial-group/tutorial-group.model';
+import { Course, Language } from 'app/entities/course.model';
+import { User } from 'app/core/user/user.model';
+import { TutorialGroupFormData } from 'app/course/tutorial-groups/tutorial-groups-instructor-view/tutorial-groups/crud/tutorial-group-form/tutorial-group-form.component';
+
+export interface GenerateExampleTutorialGroupParams {
+    id?: number;
+    title?: string;
+    capacity?: number;
+    campus?: string;
+    language?: Language;
+    additionalInformation?: string;
+    isOnline?: boolean;
+    teachingAssistant?: User;
+    isUserRegistered?: boolean;
+    numberOfRegisteredUsers?: number;
+    course?: Course;
+    teachingAssistantName?: string;
+    courseTitle?: string;
+}
+
+export const generateExampleTutorialGroup = ({
+    id = 1,
+    title = 'Example',
+    capacity = 10,
+    campus = 'Example Campus',
+    language = Language.ENGLISH,
+    additionalInformation = 'Example Additional Information',
+    isOnline = false,
+    teachingAssistant = { id: 1, login: 'Example TA' } as User,
+    isUserRegistered = false,
+    numberOfRegisteredUsers = 5,
+    course = { id: 1, title: 'Test Course' } as Course,
+    teachingAssistantName = 'Example TA',
+    courseTitle = 'Test Course',
+}: GenerateExampleTutorialGroupParams) => {
```

Comment:
Why do you not use `as TutorialGroup`?

Reply:
Good idea! Changed in 404a61f9e625e0e40fb810485d800baaea28b685